charts/zabbix/templates/deployment-zabbix-webservice.yaml

```diff
@@ -66,24 +66,38 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
```

Comment:
Won't be a conflict with line 38 if I define `extraInitContainers`?

Reply:
Hi @BGmot!

Well observed. I'll fix this.

Reply:
Hi @BGmot! 

I fixed this problem.